docs/developer/quickstart.md

```diff
@@ -153,3 +175,9 @@ make generate-njs-yaml
 Additionally, the [NJS ConfigMap Helm template](/deploy/helm-chart/templates/njs-modules.yaml) will need to be updated.
 This is currently a manual process - ensure the content in the `data` field matches that in the
 [NJS ConfigMap manifest](/deploy/manifests/njs-modules.yaml) `data` field.
+
```

Comment:
let's rename `Update NJS module ConfigMaps` to `Update Generated Manifests`, to reflect the added contents to the section. Note - this link to this section from https://github.com/nginxinc/nginx-kubernetes-gateway/blob/3166dce1ac09dfdbd3b3feab5444bafb49343783/docs/developer/implementing-a-feature.md will need to be updated too

Reply:
Good catch, thanks!